src/__tests__/utils/form-actions.test.ts

```diff
@@ -242,3 +256,38 @@ test('sendEmail - errors', async () => {
     },
   });
 });
+
+test('getTemplates', async () => {
+  setup({
+    models: {
+      TemplateStorage: {
+        list: jest.fn().mockReturnValue({ data: mockTemplates }),
+      },
+    },
+  });
+  const response = await getTemplates();
+
+  expect(response).toEqual(mockTemplates);
+});
+
+test('getTemplates - returns empty array if there are no templates', async () => {
```

Comment:
This appears to be testing an error being returned rather than no templates in storage?
For the getTemplates function I'd say you need to following tests:
- A generic success
- An error returned
- No data returned
- Scenarios to cover the parsing i.e. no templates, a list of templates including invalid ones.